### PR TITLE
change button value to Yes on No

### PR DESCRIPTION
**Pull request by vladi1337**

### CODE REVIEW

**Comment by rsasov** [comment]
Try to think for a more descriptive commit message which is wrapping up what you have done.

**Comment by atodorov** [comment]
FYI: https://chris.beams.io/posts/git-commit/

File: tcms/bugs/templates/bugs/search.html [hunk 1]
```diff
@@ -82,7 +82,7 @@
 
             <label class="col-md-1 col-lg-1" for="id_status">{% trans "Open" %}</label>
             <div class="col-md-3 col-lg-3">
-                <input class="bootstrap-switch" id="id_status" type="checkbox" checked>
+                <input class="bootstrap-switch" id="id_status" type="checkbox" checked data-on-text="Yes" data-off-text="No">
```
Comment (rsasov) [inline tcms/bugs/templates/bugs/search.html:85]: Wrap the values of the data-on-text and data-off-text attributes with trans like it's done on the line above. It's needed for internationalization.

File: tcms/bugs/templates/bugs/search.html [hunk 2]
```diff
@@ -82,7 +82,7 @@
 
             <label class="col-md-1 col-lg-1" for="id_status">{% trans "Open" %}</label>
```
Comment (rsasov) [inline tcms/bugs/templates/bugs/search.html:83]: Some changes are missing. Read again the description of your task

**Review by rsasov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**